CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ endif()
 
 project(OpenVINO DESCRIPTION "OpenVINO toolkit")
 
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "CMake build type" FORCE)
+endif()
+
```

Comment:
why this code https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/IEDevScriptsConfig.cmake#L109-L113 does not work? Looks like it's the same, but expressed via different cmake statements..

Reply:
I tried this way too. Seems like it is not included via `include(cmake/developer_package/IEDevScriptsConfig.cmake)`, also there is no 'FORCE' keyword for `set(CMAKE_BUILD_TYPE ....)`. Also it produced some other errors and my imression was that including this file is not correct solution, so including this file and making changes there are much more risky, and I've decided to `set(CMAKE_BUILD_TYPE ....)` explicitly.
But yes, if this file is necessary to be included, we should do this and fix all observed issues

Reply:
this file should be used via `find_package`, so it seems `find_package` does not propagate cache variables?